File: multimediaBack-end/multimedia/src/main/java/ao/isptec/multimedia/dto/PlaylistMusicas.java
package ao.isptec.multimedia.dto;

import java.util.List;

import ao.isptec.multimedia.model.Grupo;
import ao.isptec.multimedia.model.MembroGrupo;
import ao.isptec.multimedia.model.Musica;
import ao.isptec.multimedia.model.Playlist;
import ao.isptec.multimedia.model.PlaylistMusica;

public class PlaylistMusicas {

    private Playlist playlist;
    private List<PlaylistMusica> musicas;

    public PlaylistMusicas(Playlist playlist, List<PlaylistMusica> musicas) {
        this.playlist = playlist;
        this.musicas = musicas;
    }

    public Playlist getPlaylist() {
        return playlist;
    }

    public void setPlaylist(Playlist playlist) {
        this.playlist = playlist;
    }

    public List<PlaylistMusica> getMusicas() {
        return musicas;
    }

    public void setMusicas(List<PlaylistMusica> musicas) {
        this.musicas = musicas;
    }

}

File: isp-media-angular/src/app/models/PlaylistMusicas.ts
import { Grupo } from "./Grupo";
import { MembroGrupo } from "./MembroGrupo";
import { Musica } from "./Musica";
import { Playlist } from "./Playlist";
import { PlaylistMusica } from "./PlaylistMusica";
import { Utilizador } from "./Utilizador";

export class PlaylistMusicas {
    constructor(
        public playlist: Playlist,
        public musicas: Array<PlaylistMusica>
    ) { }
}